src/sentry/api/endpoints/organization_events_histogram.py

```diff
@@ -25,9 +25,12 @@ class HistogramSerializer(serializers.Serializer):
     def validate_field(self, fields):
         if len(fields) > 1:
             # Due to how the data is stored in snuba, multihistograms
-            # are only possible when they are all measurements.
-            if any(not is_measurement(field) for field in fields):
-                detail = "You can only generate histogram for one column at a time unless they are all measurements."
+            # are only possible when they are all measurements or all span op breakdowns.
+            if not (
+                all(is_measurement(field) for field in fields)
+                or all(is_span_op_breakdown(field) for field in fields)
+            ):
```

Comment:
Can we lift the fields check into a function, and do it with a single for loop so we don't iterate over the fields twice?
That way we can re-use the logic in `src/sentry/snuba/discover.py` we could just have the function return 
`measurement`, `op_breakdown` or `False` so its reusable